torch/autograd/profiler.py

```diff
@@ -1,30 +1,82 @@
 import itertools
-from collections import defaultdict, namedtuple
-
 import torch
 
-
-class range(object):
```

Comment:
This was a user-facing object, and shouldn't have been deleted. It's the only way to actually insert ranges from Python, which is useful.